.kokoro/client-library-check.sh

```diff
@@ -155,6 +155,8 @@ flatten-plugin)
     popd
     ;;
 *)
+    # we use the spotify version
+	sed -i 's/com.coveo:fmt-maven-plugin/com.spotify.fmt:fmt-maven-plugin/' .kokoro/build.sh
```

Comment:
The indentation is strange. Also can you add source code comment that this sed won't be needed once you fix the plugin name in build.sh?

Reply:
Could you remind me why we need to change the plugin's group id?

Reply:
> Could you remind me why we need to change the plugin's group id?

@JoeWang1127 Good question. This is because the plugin [stopped being released](https://mvnrepository.com/artifact/com.coveo/fmt-maven-plugin) from `com.coveo` in 2021. It [started being released then by spotify](https://mvnrepository.com/artifact/com.spotify.fmt/fmt-maven-plugin).

Reply:
@suztomo I corrected the indentation and added an explanation comment.

Reply:
Thank you